discovery-provider/plugins/network-monitoring/src/utils.ts

```diff
@@ -195,4 +197,31 @@ export const retryAsyncFunctionOrError = async <T>(maxTries: number, func: () =>
 
 export const asyncSleep = (milliseconds: number) => {
     return new Promise(resolve => setTimeout(resolve, milliseconds))
+}
+
+export const setupEnv = () => {
+    const nodeEnv = process.env['NODE_ENV']
+
+    if (nodeEnv === "production") {
+        console.log('[+] running in production (.env.prod)')
+        dotenv.config({ path: '.env.prod' })
+    } else if (nodeEnv === "staging") {
+        console.log('[+] running in staging (.env.stage)')
+        dotenv.config({ path: '.env.stage' })
+    } else {
+        console.log('[+] running locally (.env.local)')
+        dotenv.config({ path: '.env.local' })
+    }
+
+    // FIXME: Pull in env vars more robustly
+    const db = {
```

Comment:
is this still meant to be here?

Reply:
Yup - there's definitely still a better way of doing the env vars stuff with more validation but I can get rid of the comment if you want

Reply:
kk, can just remove comment for now